cfme/containers/project.py

```diff
@@ -36,17 +35,39 @@ def __init__(self, name, provider):
         self.name = name
         self.provider = provider
 
+    def _on_detail_page(self):
+        return sel.is_displayed(
+            '//div//h1[contains(., "{} (Summary)")]'.format(self.name))
+
     def load_details(self, refresh=False):
         if not self._on_detail_page():
             self.navigate(detail=True)
         elif refresh:
             tb.refresh()
 
+    def click_element(self, *ident):
```

Comment:
Yes, @jdemon519 I have merged #2660 which you can use to poke around. Feel free to use it.